core/nuxeo-page-provider.js

```diff
@@ -301,10 +302,7 @@ import './nuxeo-resource.js';
         method: {
           type: String,
           value() {
-            return (
-              (Nuxeo.UI && Nuxeo.UI.config && Nuxeo.UI.config.pageprovider && Nuxeo.UI.config.pageprovider.method) ||
-              'get'
-            );
+            return config.get('pageprovider.method') || 'get';
```

Comment:
would it make sense to have constants for these properties' names? should these live in the `config.js` file or in each element ?

Reply:
➕ 1 having static constants in `config.js` would make a lot of sense.

Reply:
@nelsonsilva @richardsd: how would you imagine using these in code? They can't be "constants", right? As new elements might want to register new configuration properties.

Reply:
I don't think we'll get much value out of using constants in this context, specially with this multi-level object structure.
Unless the point is to have some sort of "index" of the default config properties in this object, which could be good for documentation and DX. Then you'd use it like this?
```
return config.get(config.PAGEPROVIDER_METHOD) || 'get';
```

Reply:
I agree it only makes sense in those case where you reference the property more than once in code.